packages/extensions/core/src/lib/pipeline/pipeline.ts

```diff
@@ -359,11 +359,12 @@ export async function runPipeline(configView: AutorestContext, fileSystem: IFile
 
     // you can have --pass-thru:FOO on the command line
     // or add pass-thru: true in a pipline configuration step.
+    const configEntry = context.GetEntry(node.configScope.last.toString());
     const passthru =
-      context.GetEntry(node.configScope.last.toString())["pass-thru"] === true ||
+      configEntry?.["pass-thru"] === true ||
       values(configView.GetEntry("pass-thru")).any((each) => each === pluginName);
     const usenull =
-      context.GetEntry(node.configScope.last.toString())["null"] === true ||
+      configEntry?.["null"] === true ||
```

Comment:
I think you'll just want to reformat this file again after my last suggestion, to pass the ci.